lib/chainlock/chainlock.js

```diff
@@ -18,9 +18,9 @@ const bls = require('../crypto/bls');
  * @class ChainLock
  * @param {Buffer|Object|string} [arg] - A Buffer, Hex string, JSON string, or Object
  * representing a ChainLock
- * @property {number} height -
- * @property {Buffer} blockHash -
- * @property {Buffer} signature - merkle root of the quorum list
+ * @property {number} height
+ * @property {Buffer} blockHash
+ * @property {Buffer} signature
  */
 function ChainLock(arg) {
```

Comment:
I guess we can start to use classes since we switched ES6.

Reply:
Can we do that in a seperate PR? I also would like to change var to let/const in test for example, but don't want to put too much of syntax related stuff into this functional PR.

Reply:
Sound good!